ironfish/src/metrics/gauge.ts

```diff
@@ -0,0 +1,18 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+export class Gauge {
+  private _value: number
+
+  constructor() {
+    this._value = 0
+  }
+
+  get(): number {
+    return this._value
+  }
+
+  set(value: number): void {
+    this._value = value
```

Comment:
Can we just get rid of these getters and setters and use public value? It's the same thing with a lot less code. If we need to add these later, it would be easy to do so.